recipes/fonttools/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "fonttools" %}
+{% set version = "3.20.1" %}
+{% set sha256 = "089201e7b48f1c44e2d29663745ceeef5ef4f95454dbd3242689c77fe0fb4232" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - brotli
+    - unicodedata2
+    - munkres
+
+test:
+  imports:
+    - fonttools
+
+about:
+  home: https://github.com/fonttools/fonttools
+  license: BSD-style
```

Comment:
The `setup.py` claims this as BSD, but it looks like an old variant of the MIT license:

https://fedoraproject.org/wiki/Licensing:MIT#Old_Style_with_legal_disclaimer

Not sure if we should really change anything here. Just wanted to point it out.

Reply:
Thank you! I was rather confused by that.

Reply:
That said, I guess it's preferable to label the license as the authors of the package did. I wouldn't change this to `MIT`. It is confusing though. Maybe raise an issue seeking clarification upstream?

Reply:
for reference https://github.com/fonttools/fonttools/issues/1127